.gitignore

```diff
@@ -6,3 +6,4 @@ export_presets.cfg
 
 # Mono-specific ignores
 .mono/
+*.import
```

Comment:
What's the implication of ignoring the .import files? I could find up to date information about it, most is from 2 years ago where people just say to ignore the import folder, not both.

Reply:
Github was trying to document the .import files and this prevented that from happening